src/mmw/js/src/analyze/views.js

```diff
@@ -364,6 +368,41 @@ var PointSourceTableView = Marionette.CompositeView.extend({
 
     onAttach: function() {
         $('[data-toggle="table"]').bootstrapTable();
+    },
+
+    ui: {
+        'pointSourceId': '.point-source-id'
+    },
+
+    events: {
+        'mouseover @ui.pointSourceId': 'showPointSourceMarker',
+        'mouseout @ui.pointSourceId': 'removePointSourceMarker'
+    },
+
+    showPointSourceMarker: function(e) {
+        var data = $(e.currentTarget).data(),
+            latLng = L.latLng([data.lat, data.lng]),
+            map = App.getLeafletMap();
+
+        this.marker = L.circleMarker(latLng, {
+            fillColor: "#ff7800",
+            weight: 0,
+            fillOpacity: 0.75
+        }).bindPopup(new pointSourceLayer.PointSourcePopupView({
+          model: new Backbone.Model(data)
+        }).render().el);
+
+        this.marker.addTo(map);
+        map.panTo(this.marker.getLatLng());
```

Comment:
I think we should ditch this and let the map only show the point if it happens to be in the viewport.  I found it pretty jarring to have the map zoom around both at high and low zoom levels.  Perhaps if the link is actually clicked, we then `panTo`?